great_expectations/data_context/data_context.py

```diff
@@ -1507,13 +1512,55 @@ def does_config_exist_on_disk(cls, context_root_dir):
 
     @classmethod
     def is_project_initialized(cls, ge_dir):
```

Comment:
"initialized" sounds confusing for me - my guess would be that to be initialized a context must have the config file and the basic directory.  Maybe "activated"?